cli/scale.go

```diff
@@ -91,7 +91,12 @@ func runScale(args *docopt.Args, client *controller.Client) error {
 		if err != nil {
 			fmt.Println(commands["scale"].usage)
 		}
-		processes[arg[:i]] = val
+		processType := arg[:i]
+		if _, ok := release.Processes[processType]; ok {
+			processes[processType] = val
+		} else {
+			fmt.Printf("No process type exists with name \"%s\".\n", processType)
```

Comment:
I think this should collect the invalid process types and then exit if necessary, e.g.:

```
$ flynn scale web=1 workr=2 foo=3
ERROR: unknown process types: "workr", "foo"
```

Also, you should use `%q` rather than `\"%s\"` in the formatter string.